scripts/ci/install_breeze.sh

```diff
@@ -19,6 +19,6 @@ set -euxo pipefail
 
 cd "$( dirname "${BASH_SOURCE[0]}" )/../../"
 
-python -m pip install pipx packaging==23.1
```

Comment:
The `packageing==23.1` was only needed to workaround latest packaging breaking `pipx`,  but @uranusjr was kind enough to release 1.2.1 version of `pipx` that nicely deals with it,

Reply:
Also added `pipx>=1.2.1` in a few places (docs and scripts) to make sure the new version of `pipx` will be used across the board.